### DISCUSSION TITLE
4.4 Release

### DISCUSSION THREAD

**Discussion by hazendaz** [mathieucarbou/license-maven-plugin#759] [General & Support]
Its been some time since a release.  The following is generally what is included in this version.

license plugin
--------------
- Charset usage updates throughout
- Remove internally deprecated code
- Add to default exclusions: 
    - **/*.checkstyle
    - **/*.editorconfig
    - **/*.pmd
    - **/*.pmdruleset.xml
- General code cleanup
- Requires maven 3.6.3
- Suggest last java 8 expected release so jgit can be upgraded

build related
-------------
- no longer using super pom
- build adjustments throughout poms

**Comment by hazendaz** [comment]
@mathieucarbou Would you be able to cut a release?  All pull requests to be merged are in (including docs), milestone applied to all for 4.4, general list above for items included.  Biggest difference is likely to be release given the parent pom no longer in the builds; so this is first time a release will be done without the original parent pom.  Last release was back in September.  Biggest user change is requiring maven 3.6.3 per maven standards.  I also think this should be last java 8 based release.  Where it lands after this is probably a separate discussion but think it should just move to 17 as most things are.

**Reply by mathieucarbou** [reply]
Yes! Tomorrow ;-)

**Reply by mathieucarbou** [reply]
>  I also think this should be last java 8 based release

I also agree.
Maybe next effort could be to drop java 8 support in the code base
